lib/amazon/parser/flavor.rb

```diff
@@ -3,6 +3,8 @@ class Parser
     module Flavor
       def parse_flavors(hash, _scope)
         attributes                  = hash["product"]["attributes"]
```

Comment:
The `=` alignment with below looks weird now that there's a return in between

Reply:
this was all `rubocop -a` :-)

Reply:
no this was't, I'l fix it

Reply:
fixed